health/health.go

```diff
@@ -31,6 +31,11 @@ func (h *Handler) Add(c Checker) {
 	h.checkers = append(h.checkers, c)
 }
 
+// AddFunc adds a new check function to the handler.
+func (h *Handler) AddFunc(f CheckerFunc) {
```

Comment:
Why is this necessary, won't `Add` work directly?

Reply:
It's convenient for writing:

```go
healthHandler.AddFunc(func() error {
  // ...
})
```

instead of:


```go
healthHandler.Add(health.CheckerFunc(func() error {
  // ...
}))
```

([`net/http.ServeMux.HandleFunc`](https://godoc.org/net/http#ServeMux.HandleFunc) exists for similar reasons.)

It's a minor thing, so I'd be fine pulling this out.

Reply:
I'd suggest either leaving it out, or making it clear in the docstring that this is just a shorthand. I think I'd lean toward the former, but either way is fine.